pybamm/models/submodels/particle/size_distribution/fickian_single_distribution.py

```diff
@@ -106,18 +106,15 @@ def get_coupled_variables(self, variables):
             [self.domain.lower() + " particle size"],
         )
         T_k_xav = pybamm.PrimaryBroadcast(T_k_xav, [self.domain.lower() + " particle"],)
-        R = pybamm.PrimaryBroadcast(
-            R_spatial_variable, [self.domain.lower() + " particle"],
-        )
 
         if self.domain == "Negative":
             N_s_xav_distribution = -self.param.D_n(
                 c_s_xav_distribution, T_k_xav
-            ) * pybamm.grad(c_s_xav_distribution) / R
+            ) * pybamm.grad(c_s_xav_distribution)
         elif self.domain == "Positive":
             N_s_xav_distribution = -self.param.D_p(
                 c_s_xav_distribution, T_k_xav
-            ) * pybamm.grad(c_s_xav_distribution) / R
+            ) * pybamm.grad(c_s_xav_distribution)
```

Comment:
why this change?

Reply:
This is just to make the definition of the nondimensional flux consistent with `FickianManyParticles` (where you can have a size distribution in x), and also the scaling used for the MP-DFN in our recent JES paper. The `rhs` is unchanged, as the factor of `/ R` is just moved to lines 166 and 172, turning an `/ R` into `/ R ** 2`. Should make it easier to combine the "distribution" and "no distribution" submodels later on.

Meant to make this change much earlier, but seems I forgot. Sorry for the confusion here!